FirebaseInstallations/Source/Library/FIRSecureStorage.m

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "FIRSecureStorage.h"
+#import <Security/Security.h>
+
+#if __has_include(<FBLPromises/FBLPromises.h>)
```

Comment:
I haven't found out why Firebase imports work as is but FBLPromises doesn't. I'll investigate it. Any suggestions are appreciated.

Reply:
Weird CocoaPods should always make the module syntax work.

Reply:
I suspect, there might be some details with https://github.com/google/promises/blob/master/PromisesObjC.podspec I'll make sure it's resolved before we publish the pod